modeling/04_Per_Sat_Clean_Up.Rmd

```diff
@@ -102,25 +102,48 @@ rs_class_summ <- rs_subset %>%
   mutate_at(vars(openWater_ha, cloud_ha, lightNSSed_ha, darkNSSed_ha, offShoreSed_ha),
             ~ (./classified_ha)*100) %>%
   select(missDate, openWater_ha, cloud_ha, lightNSSed_ha, darkNSSed_ha, offShoreSed_ha) %>%
-  rename_with(~ str_replace(., "_ha$", "_perc"), ends_with("_ha"))
+  rename_with(~ str_replace(., "_ha$", "_perc_class_area"), ends_with("_ha"))
+
+rs_area_summ <- rs_subset %>%
+  mutate_at(vars(openWater_ha, cloud_ha, lightNSSed_ha, darkNSSed_ha, offShoreSed_ha),
+            ~ (./area_ha)*100) %>%
+  select(missDate, openWater_ha, cloud_ha, lightNSSed_ha, darkNSSed_ha, offShoreSed_ha) %>%
+  rename_with(~ str_replace(., "_ha$", "_perc_aoi_are"), ends_with("_ha"))
```

Comment:
It will be interesting to play with `_perc_class_area` vs `_perc_aoi_area` on my modeling end and see how that changes the picture.

```suggestion
  rename_with(~ str_replace(., "_ha$", "_perc_aoi_area"), ends_with("_ha"))
```